test/landing_page_test.dart

```diff
@@ -1,27 +1,20 @@
 import 'dart:io';
```

Comment:
Should this not be in the landing page package?

Reply:
I wanted to remove as many dependencies as possible. Landing page test is really a test of everything and by moving it here I am able to remove some cross dependencies. I would suggest maybe renaming the test? Or are we fine with having many dependencies?

Reply:
To me it looks like it really just tests only landing page. I get that it might create cross dependencies but it just looks misplaced.

Reply:
Done